LIBVmonitor0.11/LIBVmonitor0.11.ino

```diff
@@ -10,217 +10,955 @@
+	BPAPdisplay.FST = BPAPsetting.FST;
+}
 
-void BEEP(){
-  digitalWrite(BUZZER, 1);
-  delay(100);
-  digitalWrite(BUZZER, 0);
+void DISPLAY2SETTING(){
+	modeVCV = disp_modeVCV;
+	VCVsetting.IP = VCVdisplay.IP;
+	VCVsetting.TV = VCVdisplay.TV;
+	VCVsetting.RR = VCVdisplay.RR;
+	VCVsetting.IE = VCVdisplay.IE;
+	VCVsetting.PEEP = VCVdisplay.PEEP;
+
+	BPAPsetting.IP = BPAPdisplay.IP;
+	BPAPsetting.PEEP = BPAPdisplay.PEEP;
+	BPAPsetting.TP = BPAPdisplay.TP;
+	BPAPsetting.FST = BPAPdisplay.FST;
 }
 
 void EEPROM_update(){
-	EEPROM.update(TV_ADD, TV/10);
-    EEPROM.update(TI_ADD, TI/100);
-    EEPROM.update(IE_ADD, IE);
+  EEPROM.update(EEPROMaddress.MODE, (int)modeVCV);
+
+  EEPROM.update(EEPROMaddress.VCV_IP, VCVsetting.IP);
+  EEPROM.update(EEPROMaddress.VCV_TV, int(VCVsetting.TV*singleByteConst.TV));
+  EEPROM.update(EEPROMaddress.VCV_RR, VCVsetting.RR);
+  EEPROM.update(EEPROMaddress.VCV_IE, int(VCVsetting.IE*singleByteConst.IE));
+  EEPROM.update(EEPROMaddress.VCV_PEEP, VCVsetting.PEEP);
+
+  EEPROM.update(EEPROMaddress.BPAP_IP, BPAPsetting.IP);
+  EEPROM.update(EEPROMaddress.BPAP_PEEP, BPAPsetting.PEEP);
+  EEPROM.update(EEPROMaddress.BPAP_TP, BPAPsetting.TP);
+  EEPROM.update(EEPROMaddress.BPAP_FST, int(BPAPsetting.FST*singleByteConst.FST));
 }
 
 void EEPROM_read(){
-	TV = EEPROM.read(TV_ADD)*10;
-  	TI = EEPROM.read(TI_ADD)*100;
-  	IE = EEPROM.read(IE_ADD);
+  modeVCV = (bool)EEPROM.read(EEPROMaddress.MODE);
+
+  VCVsetting.IP   = EEPROM.read(EEPROMaddress.VCV_IP);
+  VCVsetting.TV   = EEPROM.read(EEPROMaddress.VCV_TV)/singleByteConst.TV;
+  VCVsetting.RR  = EEPROM.read(EEPROMaddress.VCV_RR);
+  VCVsetting.IE   = EEPROM.read(EEPROMaddress.VCV_IE)/singleByteConst.IE;
+  VCVsetting.PEEP = EEPROM.read(EEPROMaddress.VCV_PEEP);
+  
+  BPAPsetting.IP   = EEPROM.read(EEPROMaddress.BPAP_IP);
+  BPAPsetting.PEEP = EEPROM.read(EEPROMaddress.BPAP_PEEP);
+  BPAPsetting.TP   = EEPROM.read(EEPROMaddress.BPAP_TP);
+  BPAPsetting.FST  = EEPROM.read(EEPROMaddress.BPAP_FST)/singleByteConst.FST;
 }
 
 
-void wireData(uint8_t _add){
-  Wire.beginTransmission(_add);
-  Wire.write(TV/10);
-  Wire.write(TI/100);    
-  Wire.write(IE); 
+void wireData(uint8_t _addr, I2C_Func _func){
+  Wire.beginTransmission(_addr);
+  Wire.write(_func);
+
+  switch (_func) {
+      case I2C_CMD_MOTOR:     // Send motor cmd to slave
+        Wire.write((int)isBreathing);
+        break;
+
+      case I2C_DATA_SETTING:  // Send setting data to slave
+        Wire.write(modeVCV);
+        Wire.write(VCVsetting.IP);    
+        Wire.write(int(VCVsetting.TV*singleByteConst.TV));
+        Wire.write(VCVsetting.RR); 
+        Wire.write(int(VCVsetting.IE*singleByteConst.IE)); 
+        Wire.write(VCVsetting.PEEP); 
+        Wire.write(BPAPsetting.IP);
+        Wire.write(BPAPsetting.PEEP); 
+        Wire.write(BPAPsetting.TP); 
+        Wire.write(int(BPAPsetting.FST*singleByteConst.FST));
+        break;
+  }
+
   Wire.endTransmission();
 }
 
+bool inRange(int a, int x, int y){
```

Comment:
I recommend changing these parameters to be "num" "low" and "high", or "num" "min" and "max" to make it a bit more clear. 

boolean functions can also be named starting with "is" to improve readability. i.e. "isInRange()".